schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
\ No newline at end of file
+  id INT,
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you make your `id` column not null, `autoincremented` and mark the id column as `primary key` as while there may be exceptions, making the primary key column not nullable generally ensures data integrity, uniqueness, and better query performance.